Update proxmox-zfs-postinstall.sh

MAButz opened the pull request:
Hi,
this is my suggestion for commit #11

MAButz committed:
Update proxmox-zfs-postinstall.sh

MAButz commented:
#11

teissler commented:
Please change `(y/n)?` in line 170 of your change to `(y/N)?`. This way it is easier to understand that everything except "y" doesn't enables the pve-no-subscription repo.

An additional `tr '[:upper:]' '[:lower:]'` for response would also improve the scripts failure proofness.

MAButz committed:
Update proxmox-zfs-postinstall.sh

Suggested from teissler:
-change (y/n)? to (y/N)?   ;This way it is easier to understand that everything except "y" doesn't enables the pve-no-subscription repo.
-$response to lowercase  ;would also improve the scripts failure proofness.

MAButz commented:
Thanks,
changes in [https://github.com/bashclub/proxmox-zfs-postinstall/pull/13/commits/b46d8c0a3952e123a3901e19edaecf60aaf187f3](https://github.com/bashclub/proxmox-zfs-postinstall/pull/13/commits/b46d8c0a3952e123a3901e19edaecf60aaf187f3)
